Fix:

pangox.c:982:17: error: implicit declaration of function 'pango_font_metrics_new' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
pangox-fontmap.c:944:21: error: implicit declaration of function 'pango_config_key_get' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

https://gitlab.gnome.org/GNOME/pango/-/issues/644
https://gitlab.gnome.org/Archive/pangox-compat/-/issues/1
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=268301
https://bugs.freebsd.org/bugzilla/attachment.cgi?id=238694
--- pango-impl-utils.h.orig	2012-08-27 18:19:32.000000000 -0500
+++ pango-impl-utils.h	2023-07-20 22:44:21.000000000 -0500
@@ -174,6 +174,11 @@
     }
 }
 
+/* From pango-font-private.h */
+PangoFontMetrics *pango_font_metrics_new (void);
+
+/* From pango-utils-private.h */
+char *pango_config_key_get (const char *key);
 
 G_END_DECLS
 
